Lib/test/test_syntax.py

```diff
@@ -894,19 +894,19 @@
 
 >>> {1, 2, 3} = 42
 Traceback (most recent call last):
-SyntaxError: cannot assign to set display
+SyntaxError: cannot assign to set display. Maybe you meant '==' instead of '='?
 
 >>> {1: 2, 3: 4} = 42
 Traceback (most recent call last):
-SyntaxError: cannot assign to dict display
+SyntaxError: cannot assign to dict display. Maybe you meant '==' instead of '='?
```

Comment:
Probably "dict display" is an unfamiliar term (even though that's what we call it). Maybe just "dict" or "dict literal"? (And I realize that `{a: b}` is a dict display but not technically a dict literal; however I still think that dict literal would be less confusing there than dict display, given that few people know the latter term.

Reply:
> Probably "dict display" is an unfamiliar term

Agreed, I will change `dict display` for `dict literal`